drivers/client_access_control/spec/models/grda_warehouse/hud/client_visibility_legacy_spec.rb

```diff
@@ -39,7 +39,7 @@
         end
         it 'user can see all clients' do
           expect(GrdaWarehouse::Hud::Client.source.source_visible_to(user).count).to eq(4)
-          expect(GrdaWarehouse::Hud::Client.destination.destination_visible_to(user).count).to eq(4)
+          expect(GrdaWarehouse::Hud::Client.destination.destination_visible_to(user).count).to eq(3)
```

Comment:
afaict there are only 3 destination clients
```
GrdaWarehouse::Hud::Client.destination.count
> 3
```

Reply:
Something upstream must have changed.  Running this test locally, I get 4 destination clients visible to the user.

And
```
GrdaWarehouse::Hud::Client.destination.count
=> 134
```

Reply:
Reviewed with @ttoomey the `4` includes a duplicate that isn't included in the updated code.